parts/chapters/subsections/12.3/EPSDBGS.fodt

```diff
@@ -3995,8 +3995,8 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
    <text:section text:style-name="Sect1" text:name="EPSDBGS">
 
 <text:h text:style-name="P17652" text:outline-level="3"><text:bookmark-start text:name="__RefHeading___Toc347675_803326780"/><text:span text:style-name="T1">EPSDBGS – </text:span>Write <text:span text:style-name="T1">End-Point Debug Data to the DEBUG File (Multiple)</text:span><text:bookmark-end text:name="__RefHeading___Toc347675_803326780"/></text:h>
-    <text:p text:style-name="_40_TextBody">This keyword, EPSDBGS, defines the end-point debug data for multiple grid blocks that should be written to the DEBUG file (*.DBG) for when the End-Point Scaling option has been activated by the ENDSCALE keyword in the RONSPEC section.</text:p>
-    <text:p text:style-name="_40_TextBody"><text:span text:style-name="T1">See </text:span><text:a xlink:type="simple" xlink:href="#8.3.36.EPSDBGS - Define the End-Point Debug Data to the DEBUG File (Multiple)|outline" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link"><text:bookmark-ref text:reference-format="text" text:ref-name="__RefHeading___Toc238206_803326780">EPSDBGS - Write End-Point Debug Data to the DEBUG File (Multiple)</text:bookmark-ref></text:a><text:s/><text:span text:style-name="T1">in the PROPS section for afull description.</text:span></text:p>
+    <text:p text:style-name="_40_TextBody">This keyword, <text:a xlink:href="#__RefHeading___Toc238206_803326780">EPSDBGS</text:a>, defines the end-point debug data for multiple grid blocks that should be written to the <text:a xlink:href="#__RefHeading___Toc605464_3199477706">DEBUG</text:a> file (*.DBG) for when the End-Point Scaling option has been activated by the <text:a xlink:href="#__RefHeading___Toc68146_2267116897">ENDSCALE</text:a> keyword in the RONSPEC section.</text:p>
```

Comment:
False positive DEBUG. Maybe change uncapitalize?

Seems to be in other files, too. I am not intending to check all files. Hence these might remain.